docs/web_ssr/set-up-and-usage/login-and-logout.md

```diff
@@ -24,23 +28,84 @@ window.mealz.user.loadWithExternalId(userID: string, forbidProfiling: boolean).s
+:::info
+If the user is logged in at the time Mealz scripts are loaded on the page, it needs to be notified at startup, as if the user just logged in
+:::
+:::warning
+Do not forget to remove the authless-id you stored and pass in the headers as detailed in the [HTTP request headers](../main-features/pre-rendered-components#http-request-headers) section
+:::
 
 ```ts
 // Example Setup
 export class Mealz {
-  // Call this method from your app when the user logs in
+  // Call this method from your app when the user logs in or when Mealz is set up
   handleLogin(user) {
     window.mealz.user
       .loadWithExternalId(user.id, user.cookiesAccepted())
       .subscribe(() => {
         alert("User logged in on Mealz!");
       });
   }
+}
+```
 
+### Log out
+
+When your user logs out, call `window.mealz.user.reset()` to disconnect the user from Mealz. Additionally, when the user logs out, you must generate a new authless ID and pass it in the headers in the future, as detailed in the [Pre-rendered Components](../main-features/pre-rendered-components) section.
+
+Just like you inform Mealz of the user's id, if your user is logged out (when Mealz scripts are loaded or after calling `window.mealz.user.reset()`), you will need to inform Mealz of the authless-id so it can remember the unlogged user:
+
+```ts
+window.mealz.user.loadWithAuthlessId(authlessId: string, forbidProfiling?: boolean);
+```
+
+It works similarly to the method `loadWithExternalId`, except you do not need to subscribe on it, as it does not do anything asynchronous on Mealz end.
+
+```ts
+// Example Setup
+export class Mealz {
   // Call this method from your app when the user logs out
   handleLogout() {
     window.mealz.user.reset();
+    this.handleAuthless()
+  }
+
+  // Call this method from your app during Mealz setup if the user is logged out
+  handleAuthless() {
+    // ...
+    // generate the new authless-id via the route /generate-authless-token
+    // ...
+    window.mealz.user.loadWithAuthlessId(authlessId, user.cookiesAccepted());
   }
 }
 ```
+
+:::info
+  Since the `loadWithExternalId` and `loadWithAuthlessId` carry the information of the profiling permission, if the permission changes while the scripts are active, you can call `window.mealz.user.updateForbidProfiling` to update the information (see [window.mealz](../customization/window-mealz#windowmealzuser) for method usage).
+:::
+
+## When Mealz scripts are not active
+
+If your user logs in when Mealz scripts are not active, there is the question of what to do the next time our components are displayed.
+
+If at this moment on startup you do the startup "normally" and pass to us the user-id, Mealz will treat the new user as-is and not as a continuation of the previous user. Which is fine, except if your website autorises unlogged carts and if the unlogged user had already pushed some recipes to cart.
```

Comment:
```suggestion
If, at startup, you proceed normally and pass the user ID to us, Mealz will treat the new user as a distinct entity rather than a continuation of the previous session. This is generally acceptable, except if your website allows unlogged carts and the unlogged user had already added some recipes to the cart.
```